scenes/IngameScores.gd

```diff
@@ -0,0 +1,31 @@
+extends VBoxContainer
+
+const score_viewer := preload("res://scenes/performance/ingame_user_score_viewer.tscn")
+
+func reload_users(ingame_users : Dictionary) -> void:
```

Comment:
colon should stick to the variable name when defining a type (except when using :=), ingame_users: Dictionary